src/Controller/OrderController.php

```diff
@@ -175,7 +175,8 @@ public function orderCreateAction($participantId, Request $request, SessionInter
                 if ($session->get('siteType') === 'dv' && $this->siteService->isDiversionPouchSite()) {
                     $order->setType('diversion');
                 }
-                if ($session->get('siteType') === 'dv' && $this->siteService->isDiversionPouchSite() === false && $order->getCurrentVersion() > 3.1 && $order->getVersion() === null) {
+                if ($session->get('siteType') === 'dv' && $this->siteService->isDiversionPouchSite() === false && (float) $params->get('order_samples_version_dv') > 3.1) {
```

Comment:
We might want to set a default version when `order_samples_version_dv` config variable is not defined else it will throw an error.